torch/onnx/_internal/fx/exporter.py

```diff
@@ -181,13 +181,25 @@ def _fill_tensor_meta(
     expected_values: Union[torch.Tensor, Tuple[torch.Tensor, ...]],
 ):
     """Fill the meta information of onnxscript_values with that from the fx FakeTensor."""
+
+    if isinstance(expected_values, (list, tuple)) and not isinstance(
+        onnxscript_values, (list, tuple)
+    ):
+        # TODO(titaiwang): Support List of Tensors: https://github.com/microsoft/onnx-script/issues/481
+        # This is the case that fx has [tensor, tensor, ...], but onnxscript_value wrapped it as a single tensor.
+        # graph_buiding not yet support list of tensors, so we don't need to handle this case for now.
+        # eg: aten_split
+        return
+
     flat_onnxscript_values, _ = _pytree.tree_flatten(onnxscript_values)
     flat_expected_values, _ = _pytree.tree_flatten(expected_values)
     for i, (onnxscript_value, expected_value) in enumerate(
         zip(flat_onnxscript_values, flat_expected_values)
     ):
         # Only set shape for now as we don't need type information.
+        # import pdb; pdb.set_trace()
         onnxscript_value.shape = tuple(expected_value.size())
+        onnxscript_value.type = expected_value.dtype
```

Comment:
I would try just setting shape for now because we want to be able to catch type mismatches I think? onnx infer shape will fill in the type info for us. So far it works fine

Reply:
I am not sure if @wschin need this for custom Op

Reply:
ok that's right. Possible to specify if we want to add dtype then? So we only assign dtypes to custom ops

Reply:
In the future I am hoping all ops will have their opschema defined.

Reply:
It seems setting dtype would break graph somehow...